docs/content/guides/operator/archives.mdx

```diff
@@ -3,107 +3,34 @@ title: Sui Archives
 description: The archive is a historical record of all transactions on Sui. Enable archiving on your Full nodes as a best practice.
```

Comment:
Not related to this doc, but would be good to add a simple info to this doc section as well. -https://docs.sui.io/guides/operator/data-management#sui-archival-data, to remove everything after the first paragraph there, and just direct / link folks to this doc.